test/functional/ui/popupmenu_spec.lua

```diff
@@ -1866,13 +1914,16 @@ describe('builtin popupmenu', function()
           set completeopt=menu,popup
           funct Set_info()
             let comp_info = complete_info()
+            if get(comp_info, 'preview_bufnr', 0) > 0
+              call add(g:bufnrs, comp_info['preview_bufnr'])
+            endif
             if comp_info['selected'] == 2
               call nvim__complete_set(comp_info['selected'], {"info": "3info"})
             endif
           endfunc
           funct TsHl()
-            let comp_info = complete_info()
-            if get(comp_info, 'previewbufnr', 0) > 0
```

Comment:
how did this work before? was the test invalid?

Reply:
typo the field shouldbe `preview_bufnr` before test missing the ts highlight in screenshot now it have..